internal/cephfs/nodeserver.go

```diff
@@ -87,14 +88,16 @@ func (ns *NodeServer) NodeStageVolume(ctx context.Context, req *csi.NodeStageVol
 
 	volOptions, _, err := newVolumeOptionsFromVolID(ctx, string(volID), req.GetVolumeContext(), req.GetSecrets())
 	if err != nil {
```

Comment:
CI fails here:
``
rpc error: code = Internal desc = error decoding volume ID (failed to decode CSI identifier, string underflow) (pv-name)
```
`err` should be a `ErrInvalidVolID` error, it is not clear to me why `errors.As()` fails